testdata/dnn/onnx/generate_onnx_models.py

```diff
@@ -11,6 +11,7 @@
 import onnxsim
 import google.protobuf.text_format
 import io
+from typing import Optional
 
```

Comment:
In the master branch there is only one empty line here

Reply:
Thanks. Now I understand, we need to make sure that the bits of the two branches are exactly the same.
Short question, if we encounter the same thing in the future, can we do this via git? rather than manual copying.

Reply:
Forwarding this question to @alalek